### ISSUE TITLE
How to debug

### ISSUE THREAD

**Issue by veggiemonk** [HenrikJoreteg/feather-app#2]
Hi @HenrikJoreteg,

Thanks a lot for that really nice feather app. I saw it in your talk at DotJS and I agree with you 100% about minimalism. Big fan!

I quickly ran into troubles when trying to debug this app.
Do you have any recommendation? 
I am not an advanced user of webpack, do you recommend starting there or is it better to start with a more 'browser approach' like devtools ?

Cheers

**Comment by HenrikJoreteg** [comment]
Hi! :) I debug in browser for most everything. If there are syntax errors it won't build and you have to check your terminal. Otherwise devtools should be fine, no? What are you struggling with?

**Comment by veggiemonk** [comment]
Hi, thanks for answering!

Dev tools don't let me put break points because, as it turns out, the code is inlined and webpack uses `eval` to execute the file. I see now I am more struggling with webpack because I don't understand what it is doing exactly. And the fact that I don't see the original code in browser does not make it easy. I guess I have to investigate further and redo the whole experiment on my own. 

Actually what I did was, I tried to convert my redux app into the feather-app by just changing just the content of the function for the worker. So far so good. When I tried to change the view and convert the view of the redux app (already in JSX) into the feather-app, I started getting error because one of the dependency I was using is calling `window` and `document`. So the server was complaining. The problem was I didn't had a line number or files to look up. The error was just sitting there by itself in the console. So I activated the "pause on exception" which allowed me to see the inlined code. I tried a little bit more but could not put break points on inlined code. That is why I asked if there were anything special to have a nice developer experience on how to debug this app. 
I see the redux app needs more work to be adapted. It must loose some weight to become a feather-app. 

I re-watched your talk at DotJS, it came online recently. It was really refreshing to have someone stand up and ask: Do we really need all this? Maybe someone should open a fitness center for bloated apps/frameworks/libraries who want to loose weight. It's a "spa" for apps.
